### PR TITLE
Fix connection warning positioning

### LINKED ISSUE #6504: Connection warning is shown in top bar under the settings icon
## Steps to reproduce
1. Have a bad connection

### Expected behaviour
Connection warning icon is shown in a good place

### Actual behaviour
Connection warning is shown in the top bar behind the conversation icon

### PR DESCRIPTION
**Pull request by marcoambrosini**
fixes #6504 

Signed-off-by: marco <marcoambrosini@pm.me>

### CODE REVIEW

File: src/components/CallView/shared/LocalMediaControls.vue [hunk 1]
```diff
@@ -545,7 +545,8 @@ export default {
 		},
 
 		showQualityWarning() {
-			return this.senderConnectionQualityIsBad || this.qualityWarningInGracePeriodTimeout
+			return true
```
Comment (nickvergessen) [inline src/components/CallView/shared/LocalMediaControls.vue:548]: 🙈

File: src/components/CallView/shared/LocalMediaControls.vue [hunk 2]
```diff
@@ -991,11 +994,7 @@ export default {
 }
 
 .network-connection-state {
-	position: absolute;
-	bottom: 3px;
-	right: 12px;
-	width: 32px;
-	height: 32px;
+
```
Comment (nickvergessen) [inline src/components/CallView/shared/LocalMediaControls.vue:997]: If removing is intentional please delete the empty class definition

File: src/components/CallView/shared/LocalMediaControls.vue [hunk 3]
```diff
@@ -1022,4 +1021,7 @@ export default {
 		background-color: transparent;
 	}
 }
+::v-deep .trigger {
+	display: flex;
```
Comment (nickvergessen) [inline src/components/CallView/shared/LocalMediaControls.vue:1025]: Does this work on the file sidebar and is obsoleting the margin i added there for the close sidebar handle?

**Review by nickvergessen — CHANGES_REQUESTED** [review]

File: src/components/CallView/shared/LocalMediaControls.vue (hunk 3 above)
Comment (marcoambrosini) [reply src/components/CallView/shared/LocalMediaControls.vue:1025]: Not sure

File: src/components/CallView/shared/LocalMediaControls.vue [hunk 4]
```diff
@@ -990,14 +992,6 @@ export default {
 	background: linear-gradient(0deg, gray, white 36px);
 }
 
-.network-connection-state {
-	position: absolute;
-	bottom: 3px;
-	right: 12px;
-	width: 32px;
-	height: 32px;
-}
```
Comment (nickvergessen) [inline src/components/CallView/shared/LocalMediaControls.vue:999]: Bildschirmfoto von 2021-11-17 22-18-54

The spacing doesn't look right to me. the icon is more to the top and too close to the right

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**